tests/unit/test_db_sync.py

```diff
@@ -72,48 +73,83 @@ def test_column_type_mapping(self):
-        self.assertEquals(mapper(json_t_or_null), 'time')
-        self.assertEquals(mapper(json_num), 'float')
-        self.assertEquals(mapper(json_int), 'number')
-        self.assertEquals(mapper(json_int_or_str), 'text')
-        self.assertEquals(mapper(json_bool), 'boolean')
-        self.assertEquals(mapper(json_obj), 'variant')
-        self.assertEquals(mapper(json_arr), 'variant')
+        self.assertEqual(mapper(json_str), 'text')
+        self.assertEqual(mapper(json_str_or_null), 'text')
+        self.assertEqual(mapper(json_dt), 'timestamp_ntz')
+        self.assertEqual(mapper(json_dt_or_null), 'timestamp_ntz')
+        self.assertEqual(mapper(json_t), 'time')
+        self.assertEqual(mapper(json_t_or_null), 'time')
+        self.assertEqual(mapper(json_num), 'float')
+        self.assertEqual(mapper(json_int), 'number')
+        self.assertEqual(mapper(json_int_or_str), 'text')
+        self.assertEqual(mapper(json_bool), 'boolean')
+        self.assertEqual(mapper(json_obj), 'variant')
+        self.assertEqual(mapper(json_arr), 'variant')
+        self.assertEqual(mapper(json_binary), 'binary')
+
+    def test_column_trans(self):
+        """Test column transformation"""
+        trans = db_sync.column_trans
+
+        # Incoming JSON schema types
+        json_str = {"type": ["string"]}
+        json_str_or_null = {"type": ["string", "null"]}
+        json_dt = {"type": ["string"], "format": "date-time"}
+        json_dt_or_null = {"type": ["string", "null"], "format": "date-time"}
+        json_t = {"type": ["string"], "format": "time"}
+        json_t_or_null = {"type": ["string", "null"], "format": "time"}
+        json_binary = {"type": ["string", "null"], "format": "binary"}
+        json_num = {"type": ["number"]}
+        json_int = {"type": ["integer"]}
+        json_int_or_str = {"type": ["integer", "string"]}
+        json_bool = {"type": ["boolean"]}
+        json_obj = {"type": ["object"]}
+        json_arr = {"type": ["array"]}
+
+        # Getting transformations for every JSON type
+        self.assertEqual(trans(json_str), '')
+        self.assertEqual(trans(json_str_or_null), '')
+        self.assertEqual(trans(json_dt), '')
+        self.assertEqual(trans(json_dt_or_null), '')
+        self.assertEqual(trans(json_t), '')
+        self.assertEqual(trans(json_t_or_null), '')
+        self.assertEqual(trans(json_num), '')
+        self.assertEqual(trans(json_int), '')
+        self.assertEqual(trans(json_int_or_str), '')
+        self.assertEqual(trans(json_bool), '')
+        self.assertEqual(trans(json_obj), 'parse_json')
+        self.assertEqual(trans(json_arr), 'parse_json')
+        self.assertEqual(trans(json_binary), 'to_binary')
 
     def test_stream_name_to_dict(self):
         """Test identifying catalog, schema and table names from fully qualified stream and table names"""
         # Singer stream name format (Default '-' separator)
-        self.assertEquals(
+        self.assertEqual(
```

Comment:
why is it necessary?

Reply:
`assertEquals` is deprecated and unitests frameworks recommends switching to `assertEqual`.